packages/flutter/lib/src/widgets/form.dart

```diff
@@ -369,6 +397,14 @@ class FormFieldState<T> extends State<FormField<T>> with RestorationMixin {
     widget.onSaved?.call(value);
   }
 
+  /// used by [FormState] to retrieve this [FormField] value
```

Comment:
If this is isn't supposed to be used by clients, then let's make it private.

Comment should start uppercased and end with a full stop.

```suggestion
  /// Used by [FormState] to retrieve this [FormField] value.
```

Reply:
It is meant to be public, then we may need to describe what is inside of the returned `Map`.